### DISCUSSION TITLE
Add voice message settings

### DISCUSSION THREAD

**Discussion by grimhilt** [element-hq/element-meta#688] [Ideas]
#### What would you like to do?
1. Disable consecutive voice message playback
2. Disable playback of paused voice message in queue system

#### Why would you like to do it?

1. Sometimes each voice message needs its own answer, consecutive voice message playback feature forces me to pause the next audio to let me write my answer.

2. I often re play an audio (A) for a specific part, pause it and then continue to play the queue. When this queue is finished I don't expect to be listening to this paused audio (A). 

#### How would you like to achieve it?

As both of those points are features, I would like to add settings to disable them.
These could be in :
* ``Preferences > Images, GIFs, and videos`` *(which could be renamed)*
* ``Preferences > Voice Message`` *(new category)*

### Additional context

Playback of paused voice message in queue system has been considered as an issue in vector-im/element-web#20817 but it seems it is actually a feature based on vector-im/element-web#17961.
I think adding a setting for this behavior would be a good compromise.

**Comment by kittykat** [comment]
Hi, I'm going to move this issue to Element meta discussions as we need to consider it cross-platform :+1:
